### PR TITLE
Fix booleans returned when setting variants

### PR DESCRIPTION
**Pull request by ajrb**
The return values were all inverted erroneously, rather embarrassing. Not seen anyone using these return values yet, so should cause no issues.

### CODE REVIEW

**Comment by KABoissonneault** [comment]
Wouldn't it be safer to just update the comments to "True if this X had no existing variants", and changing the variable name? It's not like either way is more intuitive for this function, I think as long as the docs match the behavior, it should be good

**Comment by ajrb** [comment]
It was wrong and I fixed it. I don't feel it needs the API changing for safety however if that's the consensus I will change it.

**Comment by ajrb** [comment]
I've done as requested. 
Added method documentation for getters.
Added new block variant getter for when not iterating locations. (i.e. most mod usage)

File: Assets/Scripts/Utility/AssetInjection/WorldDataVariants.cs [hunk 1]
```diff
@@ -218,6 +235,28 @@ public static string GetBuildingVariant(ref BlockRecordKey blockRecordKey, strin
             return NoVariant;
         }
 
+        /// <summary>
+        /// Gets a variant set for the specified building, based on supplied region and location.
```
Comment (KABoissonneault) [inline Assets/Scripts/Utility/AssetInjection/WorldDataVariants.cs:239]: >Gets a variant set for the specified building"
Should that be "block"?

Comment (ajrb) [reply Assets/Scripts/Utility/AssetInjection/WorldDataVariants.cs:239]: Yep, thanks for the catch.

**Review by KABoissonneault — APPROVED** [review]

**Review by numidium — APPROVED** [review]

**Pull request merged**